src/helper/CMakeLists.txt

```
@@ -53,6 +53,11 @@ else()
     target_link_libraries(sddm-helper crypt)
 endif()
 
+if (UTMPS_FOUND)
+    target_include_directories(sddm-helper SYSTEM PUBLIC ${UTMPS_INCLUDE_DIR})
```

Not used in public headers (which sddm-helper doesn't have any of anyway), so:

```suggestion
    target_include_directories(sddm-helper SYSTEM PRIVATE ${UTMPS_INCLUDE_DIR})
```